### DISCUSSION TITLE
✨ Release Candidate -- v2.0.0rc0 ✨

### DISCUSSION THREAD

**Discussion by lwasser** [hozn/stravalib#551] [Announcements]
stravalib community -- Please test out stravalib 2.0 release candidate on your existing workflows!! you can pip install it here:

`pip install stravalib==2.0.0rc0`

I've also added a draft changelog below in case you want to see what we've changed. But stravalib now supports pydantic 2.x . We also went through the entire API and improved the strava model hierarchy to better and more closely match the strava API! Finally there is support for custom types and unit conversion! 

Many thanks to @bmeares for making their first contribution to this release ✨ and of course to @jsamoocha who made all of it happen and lead the charge!  We welcome your feedback and will continue to work on small updates to tests and docs as we find them as the changes are reviewed!!

**Full Changelog**: https://github.com/stravalib/stravalib/compare/v1.7...v2.0.0rc0

**Comment by bmeares** [comment]
Testing it out now ― so excited for this release! Thank you @lwasser and @jsamoocha!

**Reply by lwasser** [reply]
awesome!! let us know what you find @bmeares !! 🚀

**Comment by JohnScolaro** [comment]
Thanks for all the work you guys do! We appreciate it!

**Reply by lwasser** [reply]
thanks for the positive feedback @JohnScolaro  🚀
